README.md

```diff
@@ -42,7 +52,36 @@ $ manticore ./path/to/binary ++ ++  # use two symbolic strings of length two as
 
 ### API
 
-Manticore has a Python programming interface which can be used to implement custom analyses.
+Manticore has a Python programming interface which can be used to implement custom analyses and is capable of detailed verification of arbitrary properties of smart contracts via its Python API. Set starting conditions, identify symbolic transactions, then review discovered states to ensure invariants for your contract hold.
+
+```python
+from manticore.ethereum import ManticoreEVM
+contract_src="""
+contract Adder {
+    function incremented(uint value) public returns (uint){
+        if (value == 1)
+            revert();
+        return value + 1;
+    }
+}
+"""
+m = ManticoreEVM()
+
+user_account = m.create_account(balance=1000)
+contract_account = m.solidity_create_contract(contract_src,
+                                              owner=user_account,
+                                              balance=0)
+value = m.make_symbolic_value()
+
+contract_account.incremented(value)
+
+for state in m.running_states:
+    print(f"can value be 1? {state.can_be_true(value == 1)}")
```

Comment:
i think we might want to not use a f-str here. imo this gets into hard to read territory due how it's a complex expression, rather than just a simple variable name. also i expect many users will not exactly be familiar with this new python feature, which can slightly interfere with comprehension.